x-pack/plugins/observability/public/components/app/cases/case_view/index.tsx

```diff
@@ -81,7 +81,7 @@ export const CaseView = React.memo(({ caseId, subCaseId, userCanCrud }: Props) =
     [caseId, formatUrl, subCaseId]
   );
 
-  const casesUrl = getUrlForApp(CASES_APP_ID);
+  const casesUrl = `${getUrlForApp(observabilityAppId)}/cases`;
```

Comment:
> There is probably more refactoring that could have been done to the cases URL generation but I tried to make minimal changes.

I see what you mean, this is my bad. I will make a note to come back and clean these up as `formatUrl` is essentially `getUrlForApp` with extra work